OpenSim/Tools/Test/testControllers.cpp

```diff
@@ -212,7 +212,7 @@ void testPrescribedControllerOnBlock(bool disabled)
     actuatorController.setName("testPrescribedController");
     actuatorController.setActuators(osimModel.updActuators());
     actuatorController.prescribeControlForActuator(0, new Constant(controlForce));
-    actuatorController.setDisabled(disabled);
+    actuatorController.setEnabled(!disabled);
```

Comment:
Since I think we tell users to also look at our test code for examples, maybe consider changing the `disabled` variable to `enabled` to promote the change?